.github/workflows/run-tests.yml

```diff
@@ -19,14 +20,14 @@ jobs:
       # but notably it does NOT override the default PROJECT_VERSION, so the
       # farmOS Composer project 2.x branch is always used.
       - name: Build farmOS 2.x-dev Docker image
-        run: docker build --build-arg FARMOS_REPO=https://github.com/${GITHUB_REPOSITORY} --build-arg FARMOS_VERSION=${FARMOS_VERSION} -t farmos/farmos:2.x-dev docker/dev
+        run: docker build --build-arg FARMOS_REPO=https://github.com/${GITHUB_REPOSITORY} --build-arg FARMOS_VERSION=${FARMOS_VERSION} --build-arg WWW_DATA_ID=33 -t farmos/farmos:2.x-dev docker/dev
       - name: Create docker-compose.yml
-        run: curl https://raw.githubusercontent.com/farmOS/farmOS/2.x/docker/docker-compose.development.yml -o docker-compose.yml
+        run: cp docker/docker-compose.testing.yml docker-compose.yml
       - name: Start containers
-        run: sudo docker-compose up -d
-      - name: Wait 10 seconds
-        run: sleep 10
+        run: docker-compose up -d
+      - name: Wait 15 seconds
+        run: sleep 15
       - name: Run PHPUnit tests
-        run: sudo docker exec -i -u www-data farmos_www_1 phpunit --verbose --debug --group farm
+        run: docker-compose exec -u www-data -T test-runner phpunit --verbose --debug --group farm
       - name: Run PHP CodeSniffer
-        run: sudo docker exec -i -u www-data farmos_www_1 phpcs /opt/drupal/web/profiles/farm
+        run: docker-compose exec -u www-data -T www phpcs /opt/drupal/web/profiles/farm
```

Comment:
Cleaned this up by using `docker-compose exec` and removing unnecessary `sudo` usage.